### PR TITLE
Added fixes and improvements of the past month to the changelog

### PR DESCRIPTION
**Pull request by mrtnptrs**
Probably some of the fixes mentioned were caused by a commit after the
last major release and should not be mentioned at the changelog. Could
someone take a quick look at that?

### CODE REVIEW

File: distribution/changelog.txt [hunk 1]
```diff
@@ -1,18 +1,24 @@
 0.0.5 (in development)
 ------------------------------------------------------------------------
+- Feature: OpenGL renderengine added.
 - Feature: Ability to rotate map elements with the tile inspector.
-- Feature: Add cheat to disable littering.
-- Feature: Add ride console command for diagnostics and changing vehicle type.
+- Feature: Added cheat to disable littering.
+- Feature: Addded ride console command for diagnostics and changing vehicle type.
```
Comment (Goddesen) [inline distribution/changelog.txt]: There is precedent for using simple present tense "add" for features, I'd leave this as is.

File: distribution/changelog.txt [hunk 2]
```diff
@@ -1,18 +1,24 @@
...
 - Feature: Allow setting ownership of map edges.
 - Feature: Allow up to 255 cars per train.
 - Feature: Importing SV4 and SC4 files with rides.
-- Feature: Added Norwegian translation
+- Feature: Added Norwegian translation and currency.
```
Comment (Goddesen) [inline distribution/changelog.txt]: Currency was not added.

Comment (mrtnptrs) [inline distribution/changelog.txt]: Yes, it was, by you: https://github.com/OpenRCT2/OpenRCT2/pull/3787/files#diff-0fb1b1795fced5d66ecd212df165929aL26 or was it removed in a later pull request and when yes, which one?

Comment (Goddesen) [inline distribution/changelog.txt]: There is nothing about Norwegian currency in that PR. I did however alter the Swedish one slightly.

Comment (mrtnptrs) [inline distribution/changelog.txt]: My bad. Will fix it when more suggestions are made to prevent too many commits.

**Comment by mrtnptrs** [comment]
So, does someone has suggestions to remove or add something to the changelog?

**Comment by Gymnasiast** [comment]
Fixes are only mentioned if they occurred in the previous stable build. Could you check the ones you added?

File: distribution/changelog.txt [hunk 3]
```diff
@@ -1,18 +1,24 @@
 0.0.5 (in development)
 ------------------------------------------------------------------------
+- Feature: OpenGL renderengine added.
```
Comment (IntelOrca) [inline distribution/changelog.txt]: "drawing engine", not "renderengine"

File: distribution/changelog.txt [hunk 4]
```diff
@@ -22,6 +28,18 @@
 - Fix: [#3451] Renaming staff is a guest command.
 - Fix: [#3210] Scenery window scrolls too far.
 - Fix: [#3635] Inspecting sidewalk path crashes game.
+- Fix: [#2754] Dragging scrollview fails when scaled.
+- Fix: [#3859] Scenery bounding box corrected.
+- Fix: [#3858] Tooltip showed when hovering over title logo.
+- Fix: [#3856] Save Game causes Crash.
+- Fix: [#3825] Selecting players button while on a multiplayer game crashes the game.
+- Fix: [#3824] Segfault when passing invalid arguments to ride set type.
+- Fix: [#3811] Multiplayer Crash on join.
```
Comment (IntelOrca) [inline distribution/changelog.txt]: This was introduced in 0.0.5, so it can be removed.

File: distribution/changelog.txt [hunk 5]
```diff
@@ -22,6 +28,18 @@
...
 - Fix: [#3210] Scenery window scrolls too far.
 - Fix: [#3635] Inspecting sidewalk path crashes game.
+- Fix: [#2754] Dragging scrollview fails when scaled.
+- Fix: [#3859] Scenery bounding box corrected.
+- Fix: [#3858] Tooltip showed when hovering over title logo.
+- Fix: [#3856] Save Game causes Crash.
+- Fix: [#3825] Selecting players button while on a multiplayer game crashes the game.
+- Fix: [#3824] Segfault when passing invalid arguments to ride set type.
+- Fix: [#3811] Multiplayer Crash on join. 
+- Fix: [#3771] Crash when kicking player in multiplayer.
```
Comment (IntelOrca) [inline distribution/changelog.txt]: This was introduced in 0.0.5, so it can be removed.

File: distribution/changelog.txt [hunk 6]
```diff
@@ -22,6 +28,18 @@
...
+- Fix: [#3717] Tunnels incorrectly drawn when paths underneath.
```
Comment (IntelOrca) [inline distribution/changelog.txt]: This was introduced in 0.0.5, so it can be removed.

File: distribution/changelog.txt [hunk 7]
```diff
@@ -22,6 +28,18 @@
...
+- Fix: [#3734] RCT1 Import money bug.
```
Comment (IntelOrca) [inline distribution/changelog.txt]: This was introduced in 0.0.5, so it can be removed.

File: distribution/changelog.txt [hunk 8]
```diff
@@ -22,6 +28,18 @@
...
+- Fix: [#3735] Advertisement campaign window bug.
+- Fix: [#3718] Fence height incorrect.
```
Comment (IntelOrca) [inline distribution/changelog.txt]: This was introduced in 0.0.5, so it can be removed.

File: distribution/changelog.txt [hunk 9]
```diff
@@ -22,6 +28,18 @@
 - Fix: [#3451] Renaming staff is a guest command.
 - Fix: [#3210] Scenery window scrolls too far.
 - Fix: [#3635] Inspecting sidewalk path crashes game.
+- Fix: [#2754] Dragging scrollview fails when scaled.
+- Fix: [#3859] Scenery bounding box corrected.
+- Fix: [#3858] Tooltip showed when hovering over title logo.
+- Fix: [#3856] Save Game causes Crash.
+- Fix: [#3825] Selecting players button while on a multiplayer game crashes the game.
```
Comment (IntelOrca) [inline distribution/changelog.txt]: This was introduced in 0.0.5, so it can be removed.

File: distribution/changelog.txt [hunk 10]
```diff
@@ -22,6 +28,18 @@
 - Fix: [#3451] Renaming staff is a guest command.
 - Fix: [#3210] Scenery window scrolls too far.
 - Fix: [#3635] Inspecting sidewalk path crashes game.
+- Fix: [#2754] Dragging scrollview fails when scaled.
+- Fix: [#3859] Scenery bounding box corrected.
+- Fix: [#3858] Tooltip showed when hovering over title logo.
+- Fix: [#3856] Save Game causes Crash.
```
Comment (IntelOrca) [inline distribution/changelog.txt]: This was introduced in 0.0.5, so it can be removed.

File: distribution/changelog.txt [hunk 11]
```diff
@@ -1,18 +1,24 @@
...
 ------------------------------------------------------------------------
+- Feature: OpenGL renderengine added.
 - Feature: Ability to rotate map elements with the tile inspector.
 - Feature: Add cheat to disable littering.
 - Feature: Add ride console command for diagnostics and changing vehicle type.
 - Feature: Allow setting ownership of map edges.
 - Feature: Allow up to 255 cars per train.
 - Feature: Importing SV4 and SC4 files with rides.
-- Feature: Added Norwegian translation
+- Feature: Add Norwegian translation and currency.
```
Comment (Nubbie) [inline distribution/changelog.txt]: > and currency.

The currency was never added, it was a change of the existing currency in game
You should remove that as it's the swedish currency you're referring

**Comment by mrtnptrs** [comment]
@Gymnasiast I already tried to look if they were caused by a commit after the last major release with the information within the issues. Some of these issues are unique situation and almost impossible to reproduce.

**Comment by Gymnasiast** [comment]
When in doubt, you can leave them in.

**Comment by mrtnptrs** [comment]
But Ted looked at it, so many got already deleted again.

File: distribution/changelog.txt [hunk 12]
```diff
@@ -1,18 +1,24 @@
...
 - Feature: Ability to rotate map elements with the tile inspector.
 - Feature: Add cheat to disable littering.
 - Feature: Add ride console command for diagnostics and changing vehicle type.
 - Feature: Allow setting ownership of map edges.
 - Feature: Allow up to 255 cars per train.
 - Feature: Importing SV4 and SC4 files with rides.
-- Feature: Added Norwegian translation
+- Feature: Add Norwegian translation.
 - Feature: Cheat to disable plant aging.
+- Feature: Chain cheat that allows any piece to be made into a track piece.
```
Comment (Gymnasiast) [inline distribution/changelog.txt]:
```
- Feature: Cheat that allows any track piece to use a chain lift.
```

Comment (mrtnptrs) [inline distribution/changelog.txt]: Thanks you, it was difficult for me to write it down correctly :) Will fix it tonight and waiting for more suggestions. :)

**Comment by mrtnptrs** [comment]
Please don't merge it yet, I want to look at some more things.

**Comment by mrtnptrs** [comment]
Could we please have more discussion about what to do with the issue template, as how it is now, it doesn't really seem to work that great. So, how can we improve it?

**Comment by IntelOrca** [comment]
@mrtnptrs if people aren't getting the hint with the current template then they are never going to get it.

**Comment by mrtnptrs** [comment]
I already deleted my comment here, as this is the one abou the changelog :)

File: distribution/changelog.txt [hunk 13]
```diff
@@ -1,27 +1,40 @@
...
 - Fix: [#3361] Missing Twister coaster piece.
 - Fix: [#3418] Launched freefall restraints are drawn incorrectly when up (original bug).
 - Fix: [#3451] Renaming staff is a guest command.
-- Fix: [#3210] Scenery window scrolls too far.
 - Fix: [#3635] Inspecting sidewalk path crashes game.
+- Fix: [#3735] Advertisement campaign window bug.
+- Fix: [#3771] Crash when kicking player in multiplayer.
+- Fix: [#3824] Segfault when passing invalid arguments to ride set type.
+- Fix: [#3858] Tooltip showed when hovering over title logo.
+- Fix: [#3859] Scenery bounding box corrected.
```
Comment (marijnvdwerf) [inline distribution/changelog.txt]: Scenery was still handled by original code in 0.0.4.

File: distribution/changelog.txt [hunk 14]
```diff
@@ -1,27 +1,40 @@
...
 - Fix: [#3352] Assertion triggered while fixing surface tiles.
 - Fix: [#3361] Missing Twister coaster piece.
 - Fix: [#3418] Launched freefall restraints are drawn incorrectly when up (original bug).
 - Fix: [#3451] Renaming staff is a guest command.
-- Fix: [#3210] Scenery window scrolls too far.
 - Fix: [#3635] Inspecting sidewalk path crashes game.
+- Fix: [#3735] Advertisement campaign window bug.
+- Fix: [#3771] Crash when kicking player in multiplayer.
+- Fix: [#3824] Segfault when passing invalid arguments to ride set type.
+- Fix: [#3858] Tooltip showed when hovering over title logo.
```
Comment (marijnvdwerf) [inline distribution/changelog.txt]: Shown

File: distribution/changelog.txt (hunk 13 above)
Comment (Gymnasiast) [inline distribution/changelog.txt]: Indeed. This line should be removed.

File: distribution/changelog.txt [hunk 15]
```diff
@@ -1,27 +1,39 @@
...
-- Feature: Added Norwegian translation
-- Feature: Cheat to disable plant aging.
+- Feature: Add Norwegian translation.
+- Feature: Add cheat to disable littering.
+- Feature: Add Cheat to disable plant aging.
+- Feature: Add Cheat that allows any track piece to use a chain lift.
+- Feature: Add Console command to set vehicle friction.
+- Feature: Add console command to set scenario initial cash.
+- Changed: Allow raising terrain to 64 in-game units.
+- Improved: Restore horizontal and vertical scrollbar behavior from RCT2 when clicking on one of the scrollbars.
```
Comment (Gymnasiast) [inline distribution/changelog.txt]: Is a fix. The OpenRCT2 way was a bug and not deliberate .

File: distribution/changelog.txt [hunk 16]
```diff
@@ -1,27 +1,39 @@
...
 - Feature: Importing SV4 and SC4 files with rides.
-- Feature: Added Norwegian translation
-- Feature: Cheat to disable plant aging.
+- Feature: Add Norwegian translation.
+- Feature: Add cheat to disable littering.
+- Feature: Add Cheat to disable plant aging.
+- Feature: Add Cheat that allows any track piece to use a chain lift.
+- Feature: Add Console command to set vehicle friction.
+- Feature: Add console command to set scenario initial cash.
+- Changed: Allow raising terrain to 64 in-game units.
```
Comment (Gymnasiast) [inline distribution/changelog.txt]: Is a feature.

File: distribution/changelog.txt [hunk 17]
```diff
@@ -1,27 +1,39 @@
...
 - Fix: [#3361] Missing Twister coaster piece.
 - Fix: [#3418] Launched freefall restraints are drawn incorrectly when up (original bug).
 - Fix: [#3451] Renaming staff is a guest command.
-- Fix: [#3210] Scenery window scrolls too far.
 - Fix: [#3635] Inspecting sidewalk path crashes game.
+- Fix: [#3735] Advertisement campaign window bug.
+- Fix: [#3771] Crash when kicking player in multiplayer.
+- Fix: [#3824] Segfault when passing invalid arguments to ride set type.
+- Fix: [#3858] Tooltip shown when hovering over title logo.
+- Technical: [#3699] Assymmetric-key-based authorisation and assignment storage.
```
Comment (Gymnasiast) [inline distribution/changelog.txt]: Is a feature.

**Comment by Nubbie** [comment]
@mrtnptrs Status on this? :)

**Comment by mrtnptrs** [comment]
Status: can be merged when there are no more suggestions. Sorry, forgot to update it :)

**Pull request merged**